character-controller.js

```diff
@@ -480,6 +480,17 @@ function update(timeDiff) {
   localPlayer.actionInterpolants.jump.update(timeDiff);
   localPlayer.actionInterpolants.dance.update(timeDiff);
   localPlayer.actionInterpolants.throw.update(timeDiff);
+
+
+  if(localPlayer.appManager.apps[0]) // Pushing updates to physicsManager

```

Comment:
We can't assume that the `apps[0]` is the avatar. It is actually `localPlayer.avatar` which is the bound `Avatar` and you can get the `app` that of it that way. It may be `null`.